python/lsst/daf/butler/registry/tests/_registry.py

```diff
@@ -3165,3 +3165,33 @@ def testExposureQueries(self):
             ],
             [73, 72, 71, 70, 65],
         )
+
+    def test_long_query_names(self) -> None:
+        """Test that queries involving very long names are handled correctly.
+
+        This is especially important for PostgreSQL, which truncates symbols
+        longer than 64 chars, but it's worth testing for all DBs.
+        """
+        registry = self.makeRegistry()
+        name = "this_is_a_very_long_name_that_exceeds_the_postgresql_sixty_four_char_limit"
```

Comment:
I'm going to believe you, but it's not easy to verify that this is actually longer than 64 chars.

Reply:
I'll replace it.